src/vivarium_nih_us_cvd/components/interventions.py

```diff
@@ -44,6 +46,19 @@ def get_endpoint(endpoint_type: str) -> datetime:
                 endpoint = get_time_stamp(builder.configuration.time[endpoint_type])
             else:
                 endpoint = get_time_stamp(scale_up_config[endpoint_type])
-            return endpoint
+            endpoints.append(endpoint)
+
+        return endpoints[0], endpoints[1]
```

Comment:
Maybe do this as a dict instead so you can do `endpoints["start"], endpoints["end"]`. I like to steer clear of numerical indices like these if possible, since they don't convey meaning without understanding the whole context.